### DISCUSSION TITLE
Setting height of the Spectrum Combobox dropdown

### DISCUSSION THREAD

**Discussion by cheema-corellian** [adobe/react-spectrum#6500] [Q&A] [answered]
Hi folks,

I am using the Spectrum combobox component to load remote data in a manner very similar to the one shown in the async load example in the official docs here:

https://react-spectrum.adobe.com/react-spectrum/ComboBox.html#asynchronous-loading

And just like that example, my dropdown is very tall.  This creates some challenges.  Is there an easy way to limit the height of that dropdown?  I tried the built-in height prop, but that does not impact the dropdown.  I also tried wrapping it in a View component and setting the height on the View component and that didn't make a difference either.

**Comment by reidbarber** [comment, marked as answer]
There isn't a way to customize this unfortunately. We've had this question come up a few times internally, and due to Spectrum guidelines specifying that there shouldn't be a max height, we don't allow it to be customized: https://spectrum.adobe.com/page/combo-box/#Menu-height
